packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -109,6 +106,24 @@ abstract class ResidentRunner {
     await currentView.uiIsolate.flutterToggleDebugPaintSizeEnabled();
   }
 
+  Future<Null> _screenshot() async {
+    File outputFile = getUniqueFile(fs.currentDirectory, 'flutter', 'png');
+    try {
+      if (vmService != null)
+        await vmService.vm.refreshViews();
+      if (isRunningDebug)
+        await currentView.uiIsolate.flutterDebugAllowBanner(false);
```

Comment:
Should we try to extract the logic here and in screenshot.dart to share nice features like these?